src/confirm.rs

```diff
@@ -133,12 +133,11 @@ impl<'a> Confirm<'a> {
         let mut out = Buffer::ansi();
 
         out.set_color(&self.theme.title)?;
-        write!(out, " {}", self.title)?;
+        writeln!(out, " {}", self.title)?;
```

Comment:
did you want to remove the 1 column prefix?

I looked at huh and it looks like they actually put a solid line on the side which I don't really like the look of. I don't have a strong preference here, but whatever we do should be consistent across our components I think

Reply:
I guess in theory this is something that could be handled via themes if we want to go that route (not sure if huh does that or not)